bin/node/cli/src/service.rs

```diff
@@ -49,6 +54,7 @@ macro_rules! new_full_start {
 		type RpcExtension = jsonrpc_core::IoHandler<sc_rpc::Metadata>;
 		let mut import_setup = None;
 		let inherent_data_providers = sp_inherents::InherentDataProviders::new();
+		let finality_proof_subscription = grandpa::FinalityProofSubscription::new();
```

Comment:
I think that this should be part of `grandpa_link` and should be created internally by `grandpa::block_import`, so that we can have a similar setup as with `babe_link`.